src/Components/v2/Carousel.tsx

```diff
@@ -1,10 +1,17 @@
 import { Box, ChevronIcon, color, Flex, space } from "@artsy/palette"
-import { Options as FlickityOptions } from "flickity"
 import React, { Fragment } from "react"
 import styled from "styled-components"
 import { left, LeftProps, right, RightProps } from "styled-system"
 import { Media } from "Utils/Responsive"
 
+/**
+ * Do NOT use `FlickityType` to create instances of Flickity. Only refer to it
+ * for type purposes.
+ *
+ * @see BaseCarousel.prototype.componentDidMount()
+ */
+import FlickityType, { Options as FlickityOptions } from "flickity"
```

Comment:
This will not emit a top-level `require` _as long_ as it is not referred to as a runtime value.